Mvc.NoFun.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22522.0
+VisualStudioVersion = 14.0.22526.0
```

Comment:
It has to bee updated at some point. Can't avoid the churn of using daily builds. It'll quieten down eventually.